Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayerSkia.cpp

```diff
@@ -96,8 +99,10 @@ Ref<Nicosia::Buffer> CoordinatedGraphicsLayer::paintTile(const IntRect& tileRect
 
         workerPool->postTask([buffer = Ref { buffer }, recordingContext = WTFMove(recordingContext)] {
             RELEASE_ASSERT(buffer->surface());
-            GraphicsContextSkia context(sk_ref_sp(buffer->surface()), RenderingMode::Unaccelerated, RenderingPurpose::LayerBacking);
-            recordingContext->replayDisplayList(context);
+            if (auto* canvas = buffer->surface()->getCanvas()) {
```

Comment:
This is a behavioural change, isn't it? I think it's probably fine, but this changes the failure case from triggering an assertion, to silently not drawing.

Reply:
Yes, I think it's very unlikely to happen, but we don't want to crash if that's the case.